src/tenant/initialization.py

```diff
@@ -247,6 +247,7 @@ def create_initial_badges():
         active=True,
         import_id='fa3b0518-cf9c-443c-8fe4-f4a887b495a7'
     )
+    bytedeck_proficiency.tags.add("Intro")
```

Comment:
Instead of providing a string here (separate from the variable used in the other method for the same tag, can you create a variable accessible by both methods?  you can do this by creating the variable at the top of the module, outside of any methods. 

Also, please use lower case for tags "intro"

Reply:
done